bears/gettext/DennisBear.py

```diff
@@ -0,0 +1,41 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='dennis-cmd',
+        output_format='regex',
+        output_regex=r'(?P<message>(?P<severity>[EW])[0-9]{3}: .*)'
+                     r'\n(?P<line>[0-9]+):.*\n(?P<end_line>[0-9]+):.*',
+        severity_map={'W': RESULT_SEVERITY.NORMAL,
+                      'E': RESULT_SEVERITY.MAJOR})
+class DennisBear:
+    """
+    Lints your translated PO and POT files!
+
+    Check multiple lint rules on all the strings in the PO file
+    generating a list of errors and a list of warnings.
+
+    See http://dennis.readthedocs.io/en/latest/linting.html for
+    list of all error codes.
+
+    http://dennis.readthedocs.io/
+    """
+
+    LANGUAGES = {'po', 'pot'}
+    REQUIREMENTS = {PipRequirement('dennis', '0.*')}
```

Comment:
this will cause CI to fail, as this requirement now must be accurate.  So `0.8`.